checks/fuzzing_test.go

```diff
@@ -76,7 +76,7 @@ func TestFuzzing(t *testing.T) {
 			},
 			wantErr: false,
 			expected: scut.TestReturn{
-				NumberOfWarn:  6,
+				NumberOfWarn:  11,
```

Comment:
might be a good time to go back and add the filtering of negative findings to the fuzzing check, which got added in #3409
e.g for another check the code was changed to something like:
```go
for i := range findings {
		f := &findings[i]
		if f.Outcome == finding.OutcomePositive {
			// Log all findings except the negative ones.
			checker.LogFindings(nonNegative(findings), dl)
			return checker.CreateMaxScoreResult(name, "update tool detected")
		}
}
```

basically we are assuming a project only needs one fuzzing tool (this does fall apart a little for large projects)

Reply:
Am not sure I understand -- This was already added?

https://github.com/ossf/scorecard/blob/5c93fe63b4d011a5e7e299fd136ec4280479472f/checks/evaluation/fuzzing.go#L54-L56

The reason the numbers are increased in this PR is because in the relevant tests there are no success cases and that case all warnings are kept. Is there perhaps a bug here in that the second use of `LogFindings` does not sort it:

https://github.com/ossf/scorecard/blob/5c93fe63b4d011a5e7e299fd136ec4280479472f/checks/evaluation/fuzzing.go#L55-L60

Reply:
I may have just read the diff wrong. If it already calls it, then this can be resolved.